crates/cast/bin/cmd/create2.rs

```diff
@@ -124,14 +120,9 @@ impl Create2Args {
         let (salt, addr) = std::iter::repeat(())
             .par_bridge()
             .map(|_| {
-                let salt = H256::random_using(&mut thread_rng());
-                let salt = Bytes::from(salt.to_fixed_bytes());
+                let salt = B256::random();
 
-                let addr = SimpleCast::to_checksum_address(&get_create2_address_from_hash(
-                    deployer,
-                    salt.clone(),
-                    init_code_hash,
-                ));
+                let addr = SimpleCast::to_checksum_address(&deployer.create2(salt, init_code_hash));
```

Comment:
Yeah definitely. This was migrated mainly to made the code compile, but it's completely redundant now

Also agree with SimpleCast, it could just be a CLI